Description: Fix some spelling errors in manpages reported
 by lintian checks.
Origin: vendor
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=53769
Forwarded: yes
Author: Salvatore Bonaccorso <salvatore.bonaccorso@gmail.com>
--- a/lib/Sys/Virt.pm
+++ b/lib/Sys/Virt.pm
@@ -111,7 +111,7 @@
 If the optional C<readonly> parameter is supplied, then an unprivileged
 connection to the VMM will be attempted. If it is not supplied, then it
 defaults to making a fully privileged connection to the VMM. If the
-calling application is not running as root, it may be neccessary to
+calling application is not running as root, it may be necessary to
 provide authentication callbacks.
 
 If the optional C<auth> parameter is set to a non-zero value,
@@ -124,7 +124,7 @@
 types that will be supported. The credential constants in
 this module can be used as values in this list. The C<callback>
 parameter should be a subroutine reference containing the
-code neccessary to gather the credentials. When invoked it
+code necessary to gather the credentials. When invoked it
 will be supplied with a single parameter, a array reference
 of requested credentials. The elements of the array are
 hash references, with keys C<type> giving the type of
@@ -1051,7 +1051,7 @@
 
 Register a callback to received notificaitons of domain state change
 events. Only a single callback can be registered with each connection
-instance. The callback will be invoked with four paramters, an
+instance. The callback will be invoked with four parameters, an
 instance of C<Sys::Virt> for the connection, an instance of C<Sys::Virt::Domain>
 for the domain changing state, and a C<event> and C<detail> arguments,
 corresponding to the event constants defined in the C<Sys::Virt::Domain>
--- a/lib/Sys/Virt/Domain.pm
+++ b/lib/Sys/Virt/Domain.pm
@@ -392,7 +392,7 @@
 domains' current name will be maintained. In normal circumstances,
 the source host determines the target hostname from the URI associated
 with the C<destcon> connection. If the destination host is multi-homed
-it may be neccessary to supply an alternate destination hostame
+it may be necessary to supply an alternate destination hostame
 via the C<uri> parameter. The C<bandwidth> parameter allows network
 usage to be throttled during migration. If set to zero, no throttling
 will be performed. The C<flags>, C<dname>, C<uri> and C<bandwidth>
@@ -411,7 +411,7 @@
 domains' current name will be maintained. In normal circumstances,
 the source host determines the target hostname from the URI associated
 with the C<destcon> connection. If the destination host is multi-homed
-it may be neccessary to supply an alternate destination hostame
+it may be necessary to supply an alternate destination hostame
 via the C<uri> parameter. The C<bandwidth> parameter allows network
 usage to be throttled during migration. If set to zero, no throttling
 will be performed. The C<flags>, C<dname>, C<uri> and C<bandwidth>
